system/ui/widgets/keyboard.py

```diff
@@ -95,6 +102,31 @@ def render(self, title: str, sub_title: str):
 
     return -1
 
+  def _render_input_area(self, input_rect: rl.Rectangle, eye_rect: rl.Rectangle):
+    # Set password mode and render input box
+    self._input_box.set_password_mode(self._password_mode)
+    self._input_box.render(input_rect)
+    rl.draw_line_ex(
+      rl.Vector2(input_rect.x, input_rect.y + input_rect.height - 2),
+      rl.Vector2(eye_rect.x + eye_rect.width, input_rect.y + input_rect.height - 2),
+      3.0,  # 3 pixel thickness
+      rl.Color(189, 189, 189, 255),
+    )
+
+    # render eye icon
```

Comment:
Is there a way to choose not to show the visibility toggle?

Reply:
added show_password_toggle parameter to choose not to show the visibility toggle(default is not show)